README.md

```diff
@@ -24,13 +26,14 @@ Then, to build the code, run:
 For tests, ensure you have a local Postgres instance running. While in the
 `terra-data-catalog` directory, initialize the database as follows:
 
-```
+```sh
 psql -f common/postgres-init.sql
 ```
 
 After the database is initialized, then you may run integration tests as follows:
 
-```
-render-configs.sh
-./gradlew :integration:runTest --scan --args="suites/FullIntegration.json /tmp/test"
+```sh
+./gradlew bootRun & # start up a local instance of the data catalog service
```

Comment:
What does the `&` do in this command? Is it necessary?

Reply:
Yes, it's necessary so that there's an instance that runs locally so that the tests can run against it.

Reply:
I actually wouldn't recommend running it this way, since `bootRun` is very noisy. I would recommend running it in a separate terminal so you can see its output separate from the test run output.